Umee-Bots/High-Utilization-Ratio/src/agent.ts

```diff
@@ -0,0 +1,150 @@
+import BigNumber from "bignumber.js";
+import {
+  BlockEvent,
+  ethers,
+  Finding,
+  getEthersProvider,
+  HandleBlock,
+  HandleTransaction,
+  Initialize,
+  TransactionEvent,
+} from "forta-agent";
+import CONFIG from "./agent.config";
+import { GET_RESERVES_LIST_ABI, GET_RESERVE_DATA_ABI, RESERVE_INITIALIZED_ABI } from "./constants";
+import { MulticallProvider } from "./multicall";
+import {
+  AgentConfig,
+  arrayChunks,
+  createAbsoluteThresholdFinding,
+  createPercentageThresholdFinding,
+  createReserveData,
+  ReserveBalances,
+  ReserveData,
+  usageRatio,
+} from "./utils";
+
+BigNumber.set({ DECIMAL_PLACES: 18 });
+
+const reserveData: ReserveData[] = [];
+
+// testing
+export const resetReserveData = () => (reserveData.length = 0);
+export const getReserveData = () => reserveData;
+export const setReserveData = (newReserveData: ReserveData[]) => {
+  resetReserveData();
+  newReserveData.forEach((el) => reserveData.push(el));
+};
```

Comment:
I think this block should be inside agent.spec.ts since they're only related to testing

Reply:
Yes, but the idea in this case is exporting functions to access the bot local state. This could be changed to just exporting `reserveData`, but I don't know if it's worth it to change this now.